i18n/en.json

```diff
@@ -768,156 +768,167 @@
     "translation": "The settings command is not supported on your device"
   },
   {
-    "id": "api.command_shortcuts.browser.channel_next",
-    "translation": "{{.ChannelNextCmd}}: Next channel in your history\n"
+    "id": "api.command_shortcuts.cmd",
+    "translation": "Cmd"
   },
   {
-    "id": "api.command_shortcuts.browser.channel_next.cmd",
-    "translation": "ALT+RIGHT"
+    "id": "api.command_shortcuts.ctrl",
+    "translation": "Ctrl"
   },
   {
-    "id": "api.command_shortcuts.browser.channel_next.cmd_mac",
-    "translation": "CMD+]"
+    "id": "api.command_shortcuts.desc",
+    "translation": "Displays a list of keyboard shortcuts"
   },
   {
-    "id": "api.command_shortcuts.browser.channel_prev",
-    "translation": "{{.ChannelPrevCmd}}: Previous channel in your history\n"
+    "id": "api.command_shortcuts.header",
+    "translation": "Keyboard Shortcuts\n"
   },
   {
-    "id": "api.command_shortcuts.browser.channel_prev.cmd",
-    "translation": "ALT+LEFT"
+    "id": "api.command_shortcuts.nav.header",
+    "translation": "Navigation\n"
   },
   {
-    "id": "api.command_shortcuts.browser.channel_prev.cmd_mac",
-    "translation": "CMD+["
+    "id": "api.command_shortcuts.nav.prev",
+    "translation": "Previous channel:\tAlt|Up\n"
   },
   {
-    "id": "api.command_shortcuts.browser.font_decrease",
-    "translation": "{{.CmdOrCtrl}}+MINUS: Decrease font size (zoom out)\n"
+    "id": "api.command_shortcuts.nav.next",
+    "translation": "Next channel:\tAlt|Down\n"
   },
   {
-    "id": "api.command_shortcuts.browser.font_increase",
-    "translation": "{{.CmdOrCtrl}}+PLUS: Increase font size (zoom in)\n"
+    "id": "api.command_shortcuts.nav.unread_next",
+    "translation": "Next unread channel:\tAlt|Shift|Up\n"
   },
   {
-    "id": "api.command_shortcuts.browser.header",
-    "translation": "#### Built-in Browser Commands\n\n"
+    "id": "api.command_shortcuts.nav.unread_prev",
+    "translation": "Previous unread channel:\tAlt|Shift|Down\n"
```

Comment:
I think it's correct but let me know if needs to change:
`Previous unread channel: Alt + Shift + Down`